### PR TITLE
Allow multiple routes to be added for the same prefix

### PR DESCRIPTION
**Pull request by mccv1r0**
Allow multiple routes to be added for the same prefix.
Enables ECMP

Fixes: #601

### CODE REVIEW

**Comment by mccv1r0** [comment]
Formally https://github.com/containernetworking/plugins/pull/602 which entered rebase/revendor hell

File: plugins/main/bridge/bridge.go [hunk 1]
```diff
@@ -122,7 +122,7 @@ func calcGateways(result *current.Result, n *NetConf) (*gwInfo, *gwInfo, error)
 
 		// Add a default route for this family using the current
 		// gateway address if necessary.
-		if n.IsDefaultGW && !gws.defaultRouteFound {
+		if n.IsDefaultGW {
```
Comment (dcbw) [inline plugins/main/bridge/bridge.go:125]: @mccv1r0 is this to allow multiple default routes for the bridge or something?

Comment (mccv1r0) [reply plugins/main/bridge/bridge.go]: Yes, but keeping it seems to behave too, so I added `&& !gws.defaultRouteFound` back.

**Pull request merged**